res/controllers/midi-components-0.0.js

```diff
@@ -583,11 +594,16 @@
         };
         this.EffectUnitKnob.prototype = new Pot({
             group: this.group,
+            input: function (channel, control, value, status, group) {
```

Comment:
If you comment out this function it'll work without compiling #1218. It'll use Pot.prototype.input() instead.